cuda/src/fft_cuda_kernels.cu

```diff
@@ -20,13 +20,12 @@ __global__ void bit_reverse_kernel(thCdouble *out, thCdouble *in, int n)
 	int tid = threadIdx.x;
 	int offset = blockIdx.x * blockDim.x;
     int myId = tid + offset;
-    int d = log2f(n);
+    int d = roundf(log2f(n));
```

Comment:
This was the problem for 2^15! When `n = 2^15`, the value of `d` would end up as 14 instead of 15! The `roundf()` fixed that.